applications/crossbar/src/modules/cb_modules_util.erl

```diff
@@ -327,11 +331,13 @@ get_application_data_from_nouns(_Nouns) ->
 
 -spec get_timeout(cb_context:context()) -> pos_integer().
 get_timeout(Context) ->
-    try wh_util:to_integer(cb_context:req_value(Context, <<"timeout">>, 30)) of
-        Timeout when Timeout > 3 -> Timeout;
-        _ -> 30
+    Default = 30,
```

Comment:
A macro may be more descriptive `?DEFAULT_TIMEOUT_S` to indicate the unit of time is seconds here.